portfolio/src/main/webapp/style.css

```diff
@@ -91,10 +91,12 @@ a:hover {color:#560d32;}
 
 /*Secret Talents*/
 .talents-wrap {
-  align-items: center;
   display: flex;
   flex-wrap: wrap;
   justify-content: center;
+  margin-bottom: 0px;
+  margin-left: auto;
+  margin-right: auto;
```

Comment:
Use shorthand properties when possible: https://google.github.io/styleguide/htmlcssguide.html#Shorthand_Properties
And for 0 values, you can omit the units

Reply:
can I implement these changes on my next pull request? I don't want to accidentally mess something up by editing on github

Reply:
Yeah that's totally fine!